PowerEditor/src/ScintillaComponent/DocTabView.h

```diff
@@ -86,17 +72,37 @@ public :
 
 	void reSizeTo(RECT & rc) override;
 
+	void resizeIconsDpi() {
+		UINT newSize = dpiManager().scale(g_TabIconSize);
+		for (auto i : _pIconListVector)
+		{
+			ImageList_SetIconSize(i->getHandle(), newSize, newSize);
+		}
+
+		reInit();

```

Comment:
add below
```c++
if (_iconListIndexChoice != -1)
	TabBar::setImageList(_pIconListVector[_iconListIndexChoice]->getHandle());
```

Reply:
@ozone10 
Thank you for this magic line!